### ISSUE TITLE
xraydb v4.3 from PyPI does not contain tests

### ISSUE THREAD

**Issue by mrakitin** [xraypy/XrayDB#3]
Version 4.3 from [PyPI](https://files.pythonhosted.org/packages/c8/39/3ccd7d2a0cae61c4f90579ecaf6d2b4c8bf903ad4a50c7e7c3c478c4a260/xraydb-4.3.tar.gz) is missing the `tests/` dir which causes a [conda build failure](https://dev.azure.com/nsls2forge/nsls2forge/_build/results?buildId=1500&view=logs&jobId=1bf226d3-0e2f-52d8-fa93-7d9e633347b3).

The previous version 4.2 from [PyPI](https://files.pythonhosted.org/packages/7e/27/ad11ef197ba0f809e614624d2110954575a3b0624be16259e38b442b21ce/xraydb-4.2.tar.gz) has the `tests` dir. Any specific reason the tests were removed from the distribution?

```bash
$ tree xraydb-4.2
xraydb-4.2
├── LICENSE
├── MANIFEST.in
├── PKG-INFO
├── README
├── requirements.txt
├── setup.cfg
├── setup.py
├── tests
│   ├── test_chem_materials.py
│   ├── test_xray.py
│   └── test_xraydb.py
├── versioneer.py
├── xraydb
│   ├── __init__.py
│   ├── _version.py
│   ├── chemparser.py
│   ├── materials.dat
│   ├── materials.py
│   ├── utils.py
│   ├── xray.py
│   ├── xraydb.py
│   └── xraydb.sqlite
└── xraydb.egg-info
    ├── PKG-INFO
    ├── SOURCES.txt
    ├── dependency_links.txt
    ├── requires.txt
    └── top_level.txt

3 directories, 25 files
$ tree xraydb-4.3
xraydb-4.3
├── LICENSE
├── MANIFEST.in
├── PKG-INFO
├── README
├── requirements.txt
├── setup.cfg
├── setup.py
├── versioneer.py
├── xraydb
│   ├── __init__.py
│   ├── _version.py
│   ├── chemparser.py
│   ├── materials.dat
│   ├── materials.py
│   ├── utils.py
│   ├── xray.py
│   ├── xraydb.py
│   └── xraydb.sqlite
└── xraydb.egg-info
    ├── PKG-INFO
    ├── SOURCES.txt
    ├── dependency_links.txt
    ├── requires.txt
    └── top_level.txt

2 directories, 22 files
```

**Comment by newville** [comment]
@mrakitin  Oh, that's weird and sorry....  I see now that tests/ is not in MANIFEST.in.  I don't understand how the tests folder got included before.   Unfortunately PyPI does allow a re-do, so I'll push a 4.3.1 shortly, and will verify that tests is in the tarball.

**Comment by mrakitin** [comment]
No worries, and thanks for taking care of it, @newville!

**Comment by newville** [comment]
fixed in 4.3.1 (I hope!)

**Comment by mrakitin** [comment]
Great, thanks! That fixed the problem. https://github.com/nsls-ii-forge/xraydb-feedstock/pull/3 built successfully.
